aarch64-esr-web/src/lib.rs

```diff
@@ -81,8 +96,12 @@ fn show_decoded(esr: u64, decoded: Result<Vec<FieldInfo>, DecodeError>) -> Resul
     // ESR in hexadecimal
     let row = document.create_element("tr")?;
     row.set_attribute("class", "value")?;
-    let esr_hex = format!("{:016x}", esr);
-    for digit in esr_hex.chars() {
+    let esr_hex  = format!("{:016x}", esr);
+    //loop over number of chars of interset only
```

Comment:
I think you mean 'interest'? Also please format the comment the same as the others, with a capital letter and space at the start.